[Term Entry] JavaScript Dates .setUTCSeconds() | Closes Issue #2620 (#2663)
Codecademy/docs: content/javascript/concepts/dates/terms/setUTCSeconds/setUTCSeconds.md

---

> Title: '.setUTCSecond()'

Reviewer: ```suggestion
Title: '.setUTCSeconds()'
```

> Description: 'Sets the second value of a date object using UTC.'

Reviewer: ```suggestion
Description: 'Sets the second value of a date using UTC.'
```

Subjects:
  - 'Web Development'
  - 'Computer Science'
Tags:
  - 'Date'
  - 'Methods'
CatalogContent:
  - 'introduction-to-javascript'
  - 'paths/front-end-engineer-career-path'
---

The **`.setUTCSeconds()`** method sets the second value of a `Date` object according to Coordinated Universal Time(UTC) and returns the updated `Date` object.

## Syntax

> ```

Reviewer: ```suggestion
```pseudo
```

myDate.setUTCSeconds(secondsValue)
myDate.setUTCSeconds(secondsValue, millisecondsValue)
```

> The `.setUTCSeconds()` method is called on the `myDate` object with an input argument `secondsValue` that passes the new second value.

Reviewer: ```suggestion

The `.setUTCSeconds()` method is called on the `myDate` object with an input argument `secondsValue` that passes the new value for seconds.
```

Optionally, the method can also set the date's millisecond UTC values. The valid number ranges for the time units are as follows:

- **Seconds** (Required): _0_ to _59_
- **Milliseconds** (optional): _0_ to _999_

> ## Examples

Reviewer: ```suggestion
## Example
```

This example sets the second value of the `eventSeconds` object to `35`.

```javascript
const eventSeconds = new Date('2023-06-25T00:00:00');

> d.setUTCSeconds(35);

Reviewer: ```suggestion
eventSeconds.setUTCSeconds(35);
```

Reviewer: Additionally, I think it would be more complete to include a line such as:
console.log(eventSeconds.toString())

And include the output in a ```shell block

---
Title: '.setUTCSeconds()'
Description: 'Sets the second value of a date using UTC.'
Subjects:
  - 'Web Development'
  - 'Computer Science'
Tags:
  - 'Date'
  - 'Methods'
CatalogContent:
  - 'introduction-to-javascript'
  - 'paths/front-end-engineer-career-path'
---

The **`.setUTCSeconds()`** method sets the second value of a `Date` object according to Coordinated Universal Time(UTC) and returns the updated `Date` object.

## Syntax

```pseudo
myDate.setUTCSeconds(secondsValue)
myDate.setUTCSeconds(secondsValue, millisecondsValue)
```

The `.setUTCSeconds()` method is called on the `myDate` object with an input argument `secondsValue` that passes the new value for seconds.

Optionally, the method can also set the date's millisecond UTC values. The valid number ranges for the time units are as follows:

> - **Seconds** (Required): _0_ to _59_
> - **Milliseconds** (optional): _0_ to _999_

Reviewer 2: ```suggestion
- `Seconds` (Required): _0_ to _59_
- `Milliseconds` (optional): _0_ to _999_
```

## Example

This example sets the second value of the `eventSeconds` object to `35`.

> ```javascript

Reviewer 2: Can we write the result to the console and provide an output ` ```shell ` block showing the output? Like this:

````
This example results in the following output:

```shell
Output here
```
````
